l3kernel/l3sort.dtx

```diff
@@ -199,11 +199,26 @@
 % \end{variable}
 % \end{variable}
 %
-% \begin{variable}{\s_@@_mark,\s_@@_stop}
+% \begin{variable}
+%   {
+%     \s_@@_mark,
+%     \s_@@_stop,
+%     \s_@@_merge_exp_mark,
+%     \s_@@_merge_exp_stop,
+%     \s_@@_merge_exp_tail,
+%     \s_@@_merge_exp_list,
+%   }
 %   Internal scan marks.
 %    \begin{macrocode}
 \scan_new:N \s_@@_mark
 \scan_new:N \s_@@_stop
+\cs_if_exist:NT \tex_expanded:D
+  {
+    \scan_new:N \s_@@_merge_exp_mark
+    \scan_new:N \s_@@_merge_exp_stop
+    \scan_new:N \s_@@_merge_exp_tail
+    \scan_new:N \s_@@_merge_exp_list
+  }
```

Comment:
I would prefer slightly shorter names, perhaps dropping `exp`.  Same comment for most of the auxiliaries later in this file, actually.  They don't need to say quite so precisely what they do.  String space is not completely free.